eagle-xl/scripts/stellar_mass_evolution.py

```diff
@@ -0,0 +1,130 @@
+"""
+Plots the stellar mass density evolution.
+"""
+import matplotlib
+
+import unyt
+
+import matplotlib.pyplot as plt
+import numpy as np
+import sys
+import glob
+
+from swiftsimio import load
+
+from swiftsimio.statistics import SWIFTStatisticsFile
```

Comment:
Best to in the future:

`from swiftsimio import load, load_statistics`

Reply:
Traceback (most recent call last):
  File "./eagle-xl/scripts/stellar_mass_evolution.py", line 48, in <module>
    data = SWIFTStatisticsFile(stats_filename)
NameError: name 'SWIFTStatisticsFile' is not defined